packages/mdx/lib/util/resolve-evaluate-options.js

```diff
@@ -29,13 +26,13 @@
  *   this option can also be given at compile time in `CompileOptions`;
  *   you should pass this (likely at runtime), as you might get runtime errors
  *   when using `import.meta.url` / `import` / `export … from ` otherwise.
- * @property {Fragment} Fragment
+ * @property {unknown} Fragment
  *   Symbol to use for fragments (**required**).
- * @property {Jsx | null | undefined} [jsx]
+ * @property {unknown} [jsx]
  *   Function to generate an element with static children in production mode.
- * @property {JsxDev | null | undefined} [jsxDEV]
+ * @property {unknown} [jsxDEV]
  *   Function to generate an element in development mode.
- * @property {Jsx | null | undefined} [jsxs]
```

Comment:
Should we at least use something like `Function`?
We could also improve the `hast-util-to-jsx-runtime` types?
Fixing the type error is nice, but removing all type errors around this, not so sure?

Reply:
We could use `Function`, or better: `(type: any, props: any) => any`.

We should improve `hast-util-to-jsx-runtime`, but personally I think this type isn’t worth having an extra dependency regardless.

Reply:
> or better: `(type: any, props: any) => any`.

Right, and even better would be an even more correct type. We do know a lot about props and key. And at that point, it’s nice to have the more complex type defined and tested in a single place?